src/ObjectArrayStorage.php

```diff
@@ -49,7 +49,7 @@ public function get($object) : array
     {
         $values = $this->storage->get($object);
 
-        return ($values === null) ? [] : $values;
+        return $values ?? [];
```

Comment:
I don't like the null coalesce operator here, as it also allows for undefined variable without notice. This operator is equivalent to `isset($values) ? $values : []`, not strictly equivalent to the original code.